### PR TITLE
LPD-20276 Update UserAccountResourceTest

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-20276

### CODE REVIEW

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java [hunk 1]
```diff
@@ -1447,6 +1458,20 @@ private UserAccount _addAccountUserAccount(
 			accountId, userAccount);
 	}
 
+	private UserAccount _addUserAccount(
+			long siteId, AccountEntry accountEntry, UserAccount userAccount)
+		throws Exception {
+
+		userAccount = userAccountResource.postUserAccount(userAccount);
+
+		_userLocalService.addGroupUser(siteId, userAccount.getId());
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java:1467]: here we can call the other method (the one without the accountEntry) and leave here only the "connection". In this way we don't replicate the code

**Review by smottal — CHANGES_REQUESTED** [review]
@matyaswollner only a very little change. Thx

**Review by smottal — APPROVED** [review]
Good! Thx

**Comment by brbalazs** [comment]
ci:test:relevant

**Comment by matyaswollner** [comment]
ci:test:um

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:um

**Comment by smottal** [comment]
ci:stop

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:relevant

**Comment by matyaswollner** [comment]
ci:test:um

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:relevant

**Comment by matyaswollner** [comment]
ci:test:um

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**